packages/astro/src/@types/astro.ts

```diff
@@ -586,6 +586,23 @@ export interface AstroUserConfig {
 		 * ```
 		 */
 		server?: string;
+		/**
+		 * @docs
+		 * @name build.assets
+		 * @type {string}
+		 * @default `'_astro'`
+		 * @description
+		 * Specifies the directory where Astro-generated assets should live.
```

Comment:
Might want to help people understand this is within the build output rather than being able to output to `src/assets` or something. I’ve also suggested some examples for what “Astro-generated assets” are. (I’m also unsure if “generated” is the ideal word vs “processed” or “managed” or… ? But can’t make up my mind so left that as-is for now.)

```suggestion
		 * Specifies the directory in the build output where Astro-generated assets (bundled JS and CSS for example) should live.
```